packages/vue/src/uberschriftSymbol.ts

```diff
@@ -0,0 +1,5 @@
+import { InjectionKey } from "vue";
+
+export const uberschriftSymbol = Symbol("uberschriftLevel");
+
+export const uberschriftLevel = uberschriftSymbol as InjectionKey<number>;
```

Comment:
This way we have a neat typing in the inject and provide part without having to type them again and again.